deploy/rhos-dashboard.yaml

```diff
@@ -3056,7 +2225,7 @@ spec:
           {
             "allValue": null,
             "current": {
-              "selected": false,
+              "selected": true,
```

Comment:
I know it's existing, but some concerns about hard coding some of the default selections in to the dashboard itself. I feel like I addressed this in another dashboard though... but it's been a while.

Reply:
Good point, removing the saved selection results in the first value being used. We just have to remember to do that on every update now.

Reply:
I guess this is why things like grafyaml exist :)